cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
I just pass the cpplint... and this is what I get. Should they now be (un)fixed to later be (re)fixed?